libsession/src/main/java/org/session/libsession/messaging/open_groups/OpenGroupApi.kt

```diff
@@ -109,7 +109,26 @@ object OpenGroupApi {
         val defaultWrite: Boolean = false,
         val upload: Boolean = false,
         val defaultUpload: Boolean = false,
-    )
+    ) {
+        fun toPollInfo(): RoomPollInfo {
+            return RoomPollInfo(
```

Comment:
can use expression body if you like, option+enter on the `return` to let the IDE do it.

`fun toPollInfo() = RoomPollInfo(`